README.md

```diff
@@ -75,6 +102,30 @@ editorElement.disableEnvironmentBanner();
 
 For a complete version of this example, checkout this file: [public/test.html](public/test.html). It also includes another button that inserts a template in the editor to showcase the plugins.
 
+## Prosemirror
+
+The rdfa editor uses prosemirror as a base, after the `editorElement.initEditor()` function is called you will have access to the editor controller with `editorElement.controller` this is an instance of the `ProseController` class fo the [ember-rdfa-editor](https://github.com/lblod/ember-rdfa-editor)
+It provides the following methods:
+- `toggleMark(name: string, includeEmbeddedView = false)`: method which allows to enable/disable a specific mark on the current selection. Expects the name of the mark to toggle and whether the command should be applied on an embedded editor view if such a view is active.
+- `focus(includeEmbeddedView = false)`: method which allows one to focus the main editor view (or an embedded view, if such a view is active).
+- `setHtmlContent(content: string)`: sets the content of the main editor.
```

Comment:
```suggestion
- `setHtmlContent(content: string)`: sets the content of the main editor.
- htmlContent: getter property containing the "serialized" html content of the editor. This is essentially the raw content without all the plugin bells and whistles, suitable for storing in a database. It can then be loaded with the abovementioned `setHtmlContent`
```
you forgot arguably the most important one...